roles/openshift_web_console/templates/console-smoke-test-pod.yaml.j2

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: origin-web-console-smoke-test-pod
```

Comment:
Always origin? Even for the ose?

Reply:
yeah for ose we should use `ose-` as the  prefix

Reply:
thingking about it, we should name it `openshift-web-console-smoke-test-...`